_commands/basics/alias.md

```diff
@@ -0,0 +1,86 @@
+---
+---
+
+alias
+-------
+
+`alias` is a command to create shortcut to commands and applications in the terminal.
+
+~~~bash
+$ alias l="ls"
+$ l
+Documents  Downloads  Music     Public     Videos
+Desktop    Pictures   Templates
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### Usage
+Things to note: when using alias, make sure that there is no space around the equal sign.
+~~~bash
+$ alias l = "ls"
+alias l='ls -CF'
+bash: alias: =: not found
+~~~
+Another thing to note is that aliases would overwrite existing commands. For example:
+~~~bash
+$ alias ls="git status"
+$ ls
+On branch master
+Your branch is up-to-date with 'origin/master'.
+~~~
+Also, aliases are temporary to the current shell. To make the aliases permanent, please refer to the section below "Making aliases permanent".
+
+#### Example
+The above examples do not make much sense, since they simplify a two character command. The following examples show how alias can save you a lot of time.
+
+~~~bash
+$ alias s="git status"
+$ s
+On branch master
+Your branch is up-to-date with 'origin/master'.
+~~~
+This shows that you can alias commands with multiple words.
+
+#### Example
+~~~bash
+$ alias gdesk="cd ~/Desktop"
+$ pwd
+/home/user/Desktop
+~~~
+One thing to be careful about is if you alias the cd command without using the ~, your alias might not always work. For example:
+~~~bash
+$ alias gdesk="cd Desktop"
+bash: cd: Desktop: No such file or directory
+~~~
+
+### Making aliases permanent
+If you just run the alias command from the shell, it would last for the duration of the shell (until the shell is closed). To make the aliases permanent, we can just tell the shell to run these commands when the shell starts.
+
+#### Beginner
+For people who are relatively new to this, we can do this easily. Careful, for people with code already in the .bashrc file, you should instead use an editor to edit .bashrc. So, instead of "cat >" you should use "vim" or your editor of choice.
+~~~bash
+$ cat > ~/.bashrc
```

Comment:
Why don't you make this an append (`>>`) operator so it won't overwrite by default?